wifiphisher/common/phishingpage.py

```diff
@@ -336,8 +362,10 @@ def find_user_templates(self):
             # check to see if it is a directory and not in the database
             if (os.path.isdir(os.path.join(self._template_directory, name)) and
                     name not in self._templates):
-                # add it to the list
-                local_templates.append(name)
+                # check for template
+                if self.search_template(name):
+                    # add it to the list
+                    local_templates.append(name)
```

Comment:
I'm thinking for a while here. If we don't include the template without `config.ini` or `.html` wifiphisher will not display the invalid template to the users after merging this PR (If user dosen't supply the `-p` option). I think we can still include the invalid template here and add attribute (`is_valid`) to the class `Template`, and we can mark the attribute here then if the user finally choose the invalid template, we can exit program in `pywifiphisher.py` and output the error message. How do you think?

Reply:
I think that's not better way, why should we include invalid template to the program and then exit?

Reply:
If there is no warning message tool users won't know why the program doesn't include the template.
I think we can add the warning message in `search_template` and output warning message like you do before if we don't want to include this invalid template. Something like `[!] invalid template ...` (the color just follows the `pywifiphisher.py`)

Thanks.

Reply:
I'm a little lag to reply. I have no opinion for the above two solutions and you can choose the one you think is better : ) We only need to let the users know why the template cannot be used when they want to use them.

Reply:
That's probably hard way through Template object, cause we need to add attribute and  add validate, then use validate after construct...

Reply:
Sure. We can just output the invalid template message and not include them as you do before.